ReleaseNotes.md

```diff
@@ -16,6 +16,10 @@ GitHub)
 
 
 ## New Features and Enhancements:
+- Added a custom CXSMIILES feature to encode Zero order bonds. These are
```

Comment:
didn't realize this file existed! @greglandrum could we add a checklist item or something of the sort in the PR template so contributors can do the needful?

Reply:
> didn't realize this file existed! @greglandrum could we add a checklist item or something of the sort in the PR template so contributors can do the needful?

No need. I have a script that constructs these automatically from the PR/Issue titles when I prepare a release.

Reply:
@ricrogz since this section will be automatically generated, please remove this addition

Reply:
Done!